newrelic-agent/src/main/java/com/newrelic/agent/config/InfiniteTracingConfigImpl.java

```diff
@@ -19,6 +19,7 @@
     public static final String TRACE_OBSERVER = "trace_observer";
     public static final String SPAN_EVENTS = "span_events";
     public static final String FLAKY_PERCENTAGE = "_flakyPercentage";
+    public static final String FLAKY_CODE = "_flakyCode";
```

Comment:
Does the `_flakyCode` config need to be publicly documented for customer use?

Reply:
Good question! No, the headers included by `_flakyCode` and `_flakyPercentage` only impact interactions with the staging environment.